components/mastodon/sources/new-status-boosted/new-status-boosted.mjs

```diff
@@ -0,0 +1,49 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "mastodon-new-status-boosted",
+  name: "New Status Boosted",
+  description: "Emit new event when the specified status is boosted",
```

Comment:
you could also add link to the docs of the currently used endpoint on this source